### PR TITLE
fix: Fine-tuning of our CSS

### PR DESCRIPTION
**Pull request by johnathanfong**
- Responsive styling fine-tuned
- Form validations (prelim) in place @JesseNoseworthy 
- `FocusPocus` form integration cc @nsbtr

### CODE REVIEW

File: src/utils/mq.js [hunk 1]
```diff
@@ -23,13 +23,13 @@ const mq = {
...
     css`
-      @media screen and (min-width: 768px) and (max-width: 1023px) {
+      @media screen and (min-width: 768px) and (max-width: 1279px) {
         ${css(...args)};
       }
     `,
   [devices.desktop]: (...args) =>
     css`
-      @media screen and (min-width: 1024px) {
+      @media screen and (min-width: 1280px) {
```
Comment (colindamelio) [inline src/utils/mq.js]: maybe an issue with this is that 13inch monitors like mine will see the tablet as their desktop which I don't fully like.

Comment (JesseNoseworthy) [reply src/utils/mq.js]: `1400px` has been very fruitful for me on other applications.

Comment (johnathanfong) [reply src/utils/mq.js]: @JesseNoseworthy you want to increase desktop min to `1400px`? and @colindamelio you want to go back down to `1024px`?

Comment (johnathanfong) [reply src/utils/mq.js]: https://github.com/colindamelio/saraswati/pull/18/commits/a91ad2efcf9e3d086744063411e8fbcd2e0158c5 fixes my concerns re: `landingIntro` being cut-off at smaller breakpoints. `100vh` was not enough for `1024px` but I've just imposed a `min-height` value that _ensures_ we'll never have our content overflow cut off. 

cc @colindamelio @JesseNoseworthy

**Pull request merged**